Jenkinsfile

```diff
@@ -37,4 +37,12 @@ node('docker') {
             archiveArtifacts artifacts: 'artifacts/*.xz' 
         }
     }
+    docker.image('kuksa-val-dev:ubuntu20.04').inside("-v /var/run/docker.sock:/var/run/docker.sock"){ 
+        stage('Test') {
+            sh '''
+                cd /kuksa.val/build
+                # TODO and you may need sudo right for testing ctest --build-config Debug --output-on-failure --parallel 8
```

Comment:
was your intention to resolve this TODO in later PR?

Reply:
I think so. This PR let test cases building automatically. And the test cases will be also executed, after all test cases show positive.